frontend/src/components/marketplace/ControlledCarousel.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+import Carousel from 'react-bootstrap/Carousel'
+import ItemCard from '../common/ItemCard'
+
+const RECOMMANDATION_STANDARD = 1
+
+function ControlledCarousel (prop) {
+  const [index, setIndex] = useState(0)
+
+  const handleSelect = (selectedIndex, e) => {
+    setIndex(selectedIndex)
+  }
+
+  const items = prop.items
+
+  const itemList = items.map((item) => {
+    if (!item.numberOfTimesBorrowed || item.numberOfTimesBorrowed < RECOMMANDATION_STANDARD) {
+      return null
+    }
+    return <Carousel.Item><ItemCard key={item._id}
+      id={item._id}
+      borrow
+      description={item.description}
+      image={item.image}
+      name={item.name}
+      status={item.status}
+      ratingComments={item.ratingComments}
+      type={item.type}
+      rating={item.rating}
+      location={item.owner.location}/>
```

Comment:
We should show the number of times borrowed on the card so people understand why it's there, just need to pass in the `numberOfTimesBorrowed` prop

Reply:
I decided to do this just for testing purpose: description={item.numberOfTimesBorrowed}
I don't think it it worth it to add it to the ItemCard.

Reply:
@cailors could we add it conditionally just to the featured cards on the carousel so people understand why the cards are there?